docker/linux-build.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Can you document this bash file and the dockerfile a little? What's their purpose and how are they different than the dockerfile we already have?

Reply:
Sure, where I should document it? Long story short allow me to write this kind of dockerfile very simple: https://github.com/tonyredondo/xunithelloworld/blob/master/Dockerfile#L2

Reply:
A few comments right in the file would be fine. Thanks!

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/802/commits/2b2ee99680880c3f41467916e08fecbb2deb93dd

Reply:
Thanks!